uberpoet/filegen.py

```diff
@@ -88,6 +95,30 @@
 """
 
 
+def get_import_func_calls(language, import_list, indent=0):
+    out = []
+    for i in import_list:
+        if type(i) is str:
+            continue
+        module = first_in_dict(i)
+        out_language = module["language"]
+        for file_result in module["files"].values():
+            for class_num, func_nums in file_result.classes.items():
+                for func_num in func_nums:
+                    if language == Language.SWIFT:
+                        # We are a Swift module calling into a Swift module.
+                        if out_language == Language.SWIFT:
+                            text = swift_func_call_to_swift_template.format(class_num, func_num)
+                        # We are a Swift module calling into an ObjC module.
+                        elif out_language == Language.OBJC:
+                            text = swift_func_call_to_objc_template.format(class_num, func_num)
+
```

Comment:
there is a conditional missing here that I did not add yet as I needed input.

If we are an ObjC module, calling into Swift is not possible because our current function generation for Swift uses generics. 

I could either:

a) Remove the generics from Swift function gen
b) Add new and additional ObjC friendly methods that do not include generics

What do you think?

Reply:
I think B, because generics are there to exercise the compiler more.  Every specialized generic creates new code AFAIK.

Reply:
sounds good will do!